src/storybook/store/StoryBookSDK.ts

```diff
@@ -50,6 +52,25 @@ export class StoryBookSDK implements IHMSActions {
     });
   }
 
+  preview(config: HMSConfig) {
+    if (!config.authToken) {
+      this.log('invalid params');
+      return;
+    }
+    this.log('User called preview');
+    this.store.setState(store => {
+      store.room.roomState = HMSRoomState.Preview;
+      const newPeer: HMSPeer = {
+        name: config?.userName,
+        isLocal: true,
```

Comment:
another local user will get pushed in join too.

Reply:
also in join this is being done - store.room.isConnected = true; should we set hmsstate there too apart from isconnected?

Reply:
Changed it to check for existing peer before adding new one.